HelloWorldServlet/src/example/HelloWorld.java

```diff
@@ -0,0 +1,54 @@
+package example;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class HelloWorld
+ */
+@WebServlet("/SimplExample")//servlet url
+public class HelloWorld extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+       
+    public HelloWorld() {
+        super();
+        // TODO Auto-generated constructor stub
+    }
+
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		
+		response.setContentType("text/html");
```

Comment:
This logic of sending response to user is duplicated several times, I think it is better to create helper method for this named like: sendResponse

Reply:
hmm... Can you add some example to your comment, what exactly you are planing to move?

Reply:
Of course, we can extract to helper method all the logic of response, example: 
```
private void sendResponse(String message, HttpServletRequest request, HttpServletResponse response)
{	
     response.setContentType("text/html");
     ...another code...
}
```